opentelemetry-otlp/Cargo.toml

```diff
@@ -78,7 +78,7 @@ openssl = ["grpcio/openssl"]
 openssl-vendored = ["grpcio/openssl-vendored"]
 
 # http binary
-http-proto = ["prost", "opentelemetry-http", "opentelemetry-proto/gen-tonic", "http", "trace"]
+http-proto = ["prost", "opentelemetry-http", "opentelemetry-proto/gen-tonic-messages", "http", "trace"]
```

Comment:
With the changes you've made in `-proto`, I think all mentions of `gen-tonic` in `-otlp` should start using `gen-tonic-messages`.

Reply:
Which other mentions do you mean? `grpc-tonic` needs `gen-tonic` because it uses the tonic client.